Workaround for "error: redefinition of 'ROOT_CLIP_CHAIN'" with cbindgen > 0.23.

https://hg.mozilla.org/mozilla-central/diff/51947744ce12247f378a1db2379ffaad3fcd18c3/gfx/webrender_bindings/webrender_ffi.h
https://bugzilla.mozilla.org/show_bug.cgi?id=1773259
https://github.com/eqrion/cbindgen/issues/767#issuecomment-1153125949

--- a/gfx/webrender_bindings/webrender_ffi.h
+++ b/gfx/webrender_bindings/webrender_ffi.h
@@ -73,8 +73,6 @@ struct WrPipelineInfo;
 struct WrPipelineIdAndEpoch;
 using WrPipelineIdEpochs = nsTArray<WrPipelineIdAndEpoch>;
 
-const uint64_t ROOT_CLIP_CHAIN = ~0;
-
 }  // namespace wr
 }  // namespace mozilla
 
